src/store/modules/user.js

```diff
@@ -274,6 +274,16 @@ const user = {
           reject(error)
         })
       })
+    },
+    RefreshFeatures ({ commit }) {
+      return new Promise((resolve, reject) => {
+        api('listCapabilities').then(response => {
+          const result = response.listcapabilitiesresponse.capability
+          commit('SET_FEATURES', result)
+        }).catch(error => {
```

Comment:
@Pearl1594 Would not resolving the promise cause any issues ? (Promise state, GC, etc)